pkg/virt-api/rest/console.go

```diff
@@ -107,8 +107,12 @@ func (t *Console) Console(request *restful.Request, response *restful.Response)
 
 	dstAddr := string(pods.Items[0].Status.PodIP)
 
-	// FIXME, don't hardcode virt-handler port. virt-handler should register itself somehow
-	port := "8185"
+	port := string(pods.Items[0].Annotations["virtHandlerPort"])
```

Comment:
IMhO, just pass the port as an ENV var to virt-handler. Have this code parse the value of the port env var from the pod spec.